### DISCUSSION TITLE
Python package

### DISCUSSION THREAD

**Discussion by parsariyahi** [curl/trurl#108] [Ideas]
it would be very useful if we have a python package that uses this trurl.

I don't know if we have something like this in python

**Comment by dfandrich** [comment]
pycurl seems like a natural place to find a Python binding, but that project
limits itself to the easy and multi interfaces. If curl_url_* isn't going to be
exposed there, someone could create a Python binding just for that. I doubt
that's something that will be done as part of curl, though, which only supplies
a C API.
